### PR TITLE
 sequelizemm library added to resource page.

### PR DESCRIPTION
**Pull request by hasinoorit**
sequelizemm library added to resource page.

### CODE REVIEW

**Comment by WikiRik** [comment]
Thanks for this! Could you also include it in the v7 docs; https://github.com/sequelize/website/blob/main/docs/other-topics/resources.md ?

**Comment by hasinoorit** [comment]
> Thanks for this! Could you also include it in the v7 docs; https://github.com/sequelize/website/blob/main/docs/other-topics/resources.md ?

Thanks for the idea. I have added support for v7, then added the link to the v7 resource page.

File: docs/other-topics/resources.md [hunk 1]
```diff
@@ -29,6 +29,7 @@ A curated list of awesome projects surrounding Sequelize.
 * [umzug](https://github.com/sequelize/umzug) - framework-agnostic migration tool for Node.
 * [sequelize-cli](https://github.com/sequelize/cli) - The Sequelize Command Line Interface. Includes umzug-based migrations.
 * [sequelize-mig](https://github.com/MRVMV/sequelize-mig) - Sequelize migration generator
+* [sequelizemm](https://github.com/hasinoorit/sequelizemm) CLI tool to generate migration script from models within minute.
```
Comment (WikiRik) [inline docs/other-topics/resources.md:32]:
```suggestion
* [sequelizemm](https://github.com/hasinoorit/sequelizemm) - CLI tool to generate a migration script from models
```

File: versioned_docs/version-6.x.x/other-topics/resources.md [hunk 2]
```diff
@@ -56,6 +56,7 @@ title: Resources
 ### Migrations
 
 * [umzug](https://github.com/sequelize/umzug)
+* [sequelizemm](https://github.com/hasinoorit/sequelizemm) CLI tool to generate migration script from models within minute.
```
Comment (WikiRik) [inline versioned_docs/version-6.x.x/other-topics/resources.md:59]:
```suggestion
* [sequelizemm](https://github.com/hasinoorit/sequelizemm) - CLI tool to generate a migration script from models
```

**Review by WikiRik — COMMENTED** [review]
Small textual suggestion

**Review by WikiRik — APPROVED** [review]

**Pull request merged**